src/py/flwr/server/state/in_memory_state.py

```diff
@@ -89,11 +97,17 @@ def get_task_ins(
     def store_task_res(self, task_res: TaskRes) -> Optional[UUID]:
         """Store one TaskRes."""
 
+        # Validate task
+        errors = validate_task_ins_or_res(task_res)
+        if any(errors):
+            log(ERROR, errors)
+            return None
+
         # Create and set task_id
         task_id = uuid4()
         task_res.task_id = str(task_id)
```

Comment:
Should we move this down a few lines to improve consistency?

Reply:
done